app/championship/knights.py

```diff
@@ -0,0 +1,51 @@
+class Knight:
+
+    knights = dict()
+
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            protection: int
+    ) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.protection = protection
+
+    @classmethod
+    def add_knights(cls, knight_stat: dict) -> None:
```

Comment:
You add one knight in this method; rename it to `add_knight` so naming would not be misleading.